<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_register</class>
 <widget class="QDialog" name="Dialog_register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_Num_regi">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Your number is: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>211</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter your name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label__pass_regi">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_regi">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>290</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tourist_regi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>111</width>
     <height>24</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>Tourist</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name_regi">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pass_regi">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num_regi">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_regi</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_register</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>475</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
